### DISCUSSION TITLE
Tradfri power-on setting after power outage?

### DISCUSSION THREAD

**Discussion by Letterman17** [Koenkk/zigbee2mqtt#9191] [Devices]
All my Tradfri light bulbs turn on when power is restored after a power outage.
I believe Ikea may now have implementing a "power-on" setting that remembers the last known power state and restores the light bulbs to this state after a power outage (ie if the bulb was off before the outage, it'll remain off).

Very handy for bedroom lights at 2am....

Is there any way to control this setting via zigbee2mqtt?

Thank you!

**Comment by salvovitale** [comment]
In the device, under 'Exposes' you can set that behaviour.  See pic attached.

![Screenshot_2021-10-28_at_13_55_22](https://user-images.githubusercontent.com/6718364/139267031-6d07a0de-b68b-45e3-9864-e691e6784da0.png)
